package.json

```diff
@@ -77,7 +78,9 @@
     "typescript": "^3.6.3"
   },
   "resolutions": {
-    "@lerna/**/@lerna/collect-updates": "https://github.com/babel/lerna.git#babel-collect-updates"
+    "@lerna/**/@lerna/collect-updates": "https://github.com/babel/lerna.git#babel-collect-updates",
+    "#todo: remove eslint-scope resolution when babel-eslint 11.0.0-beta.2 is published": "",
```

Comment:
AFAIK, comments aren't supported

Reply:
It looks like comment but semantically it is a long sentence key.